mlrun/feature_store/feature_vector.py

```diff
@@ -112,6 +112,7 @@ def __init__(
         label_column=None,
         stats=None,
         preview=None,
+        index_keys=None,
```

Comment:
Why you're not adding it as the last kwarg ?
I don't think it will affect anything but that is a backwards incompatible change